aws/table_aws_ssm_association.go

```diff
@@ -155,9 +156,17 @@ func tableAwsSSMAssociation(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 				Hydrate:     getAwsSSMAssociation,
 			},
+			{
+				Name:        "compliance_items",
+				Description: "List of compliance information for the associated instances.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getComplianceItems,
+				Transform:   transform.FromValue(),
+			},
 			{
 				Name:        "targets",
-				Description: "A cron expression that specifies a schedule when the association runs.",
+				Description: "The instances targeted by the request to create an association.",
+				Hydrate:     getAwsSSMAssociation,
```

Comment:
Did this column not hydrate properly before we added this line?

Reply:
earlier from list call the values for target were not populated while testing the compliance_items, that's why I had updated it . However now I am getting the values for target from list , probably some issue that time. removed.